develop/api_guc.rst

```diff
@@ -336,30 +339,82 @@ Note that it may be useful to use :code:`error` during development testing, and
   HINT:  Queries are split to multiple tasks if they have to be split into several queries on the workers.
   STATEMENT:  select * from foo;
 
-Real-time executor configuration
-$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
+Adaptive executor configuration
+$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
 
-The Citus query planner first prunes away the shards unrelated to a query and then hands the plan over to the real-time executor. For executing the plan, the real-time executor opens one connection and uses two file descriptors per unpruned shard. If the query hits a high number of shards, then the executor may need to open more connections than max_connections or use more file descriptors than max_files_per_process.
+citus.executor_slow_start_interval (integer)
```

Comment:
If there is no alpha-numeric (or anything else) ordering in the names, I'd suggest to start with `max_adaptive_executor_pool_size ` which is the most important setting.